src/oidc_provider_mock/__main__.py

```diff
@@ -84,6 +84,11 @@
     multiple=True,
     type=str,
 )
+@click.option(
+    "--user-claims-file",
+    help="A JSON file with a list of predefined user claims objects.",
+    type=str,
```

Comment:
You can use `click.File()`[https://click.palletsprojects.com/en/stable/handling-files/#file-arguments] which provides better UX